sites/fast-website/src/app/css/style.css

```diff
@@ -103,7 +103,7 @@ li {
 }
 
 site-navigation ::part(content) {
-  color: var(--neutral-foreground-rest);
+  color: var(--neutral-foreground-hint);
 }
```

Comment:
Should we make this change in `site-navigation`? We also need to update the hover and focus colors here as well.